### PR TITLE
feat(53): Added validation for SignIn|SignUp screens (#53)

### PR DESCRIPTION
**Pull request by 0to1to0to1**

### CODE REVIEW

**Review by hromogen — APPROVED** [review]

File: packages/mobile/src/pages/ScreensAuthorization/SignIn/SignIn.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+import { InterfaceStore } from './../../../store';
+import { AUTH_LOGIN, PayloadAuthLogin } from './../../../store/auth/actions';
+import { SignInState, AuthorizationProps } from './../interfaces';
+import { validateInput, hideMessage } from './../helpers';
+import styles from './styles';
+import { CustomButton } from './../../../components/Button';
+import { CustomInput } from './../../../components/Input';
+import { Divider } from './../../../components/Divider';
+import { SocialLogin } from './../../../components/SocialLogin';
+import { NotifyMessage } from './../../../components/NotifyMessage';
```
Comment (dartwlad) [inline packages/mobile/src/pages/ScreensAuthorization/SignIn/SignIn.tsx:22]: Can we avoid such './../../../' imports somehow?

**Review by dartwlad — APPROVED** [review]
As for me, it's ok, but I would suggest rewriting everything with react hooks.

File: packages/mobile/src/pages/ScreensAuthorization/SignIn/SignIn.tsx (hunk 1 above)
Comment (artbitbyte) [reply packages/mobile/src/pages/ScreensAuthorization/SignIn/SignIn.tsx:22]: Yes, I will fix imports like this in future requests.

**Comment by artbitbyte** [comment]
> As for me, it's ok, but I would suggest rewriting everything with react hooks.

What is reason for this?

**Pull request merged**